### PR TITLE
fix: active category and sidebar item, video responsiveness, like count

### PR DESCRIPTION
**Pull request by JantuDeb**
# Refractor
- fixed view count and like count
- active category style
- active sidebar item style
- responsiveness to the video player
image

### CODE REVIEW

File: src/context/liked-videos/LikedVideoContext.jsx [hunk 1]
```diff
@@ -58,6 +60,7 @@ const LikedVideoProvider = ({ children }) => {
           type: REMOVE_FROM_LIKE,
           payload: { _id: data.like.video },
         });
+      console.log(data.like.video);
```
Comment (sushil-mod) [inline src/context/liked-videos/LikedVideoContext.jsx:63]: if possible remove console log

File: src/context/liked-videos/LikedVideoContext.jsx [hunk 2]
```diff
@@ -58,6 +60,7 @@ const LikedVideoProvider = ({ children }) => {
           type: REMOVE_FROM_LIKE,
           payload: { _id: data.like.video },
```
Comment (mariyasada) [inline src/context/liked-videos/LikedVideoContext.jsx:61]: remove console if not necesary

**Review by mariyasada — APPROVED** [review]
this code looks fine. remove the console from some files if not necessary

File: src/context/liked-videos/LikedVideoContext.jsx [hunk 3]
```diff
@@ -42,6 +42,8 @@ const LikedVideoProvider = ({ children }) => {
             ...data.like.video,
           },
         });
+
+      console.log(data.like.video);
```
Comment (sushil-mod) [inline src/context/liked-videos/LikedVideoContext.jsx:46]: here too

**Review by sushil-mod — APPROVED** [review]
Overall good and clean code . i liked how used useeffects for get requests .

File: src/context/liked-videos/LikedVideoContext.jsx (hunk 2 above)
Comment (JantuDeb) [reply src/context/liked-videos/LikedVideoContext.jsx:61]: Thanks will remove it

**Pull request merged**